docker/ci/scripts/prepare_ui_tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Script for preparing ui tests within a docker container. Used by ui_tests.sh.
+
+source docker/ci/scripts/prepare_tests_common.sh
+
+ulimit -n 4096
+
+export CI_JOB=ui_tests
+
+# set up locals.yml
```

Comment:
nit: Can this comment somehow imply that we are _appending_ to locals.yml which was initialized in the prepare_tests_common script?

Reply:
Good call!